pkg/kubernetes/status.go

```diff
@@ -0,0 +1,98 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package kubernetes
+
+import (
+	"fmt"
+	"strings"
+	"sync"
+
+	"github.com/dapr/cli/pkg/age"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+)
+
+var (
+	controlPlaneLabels = []string{"dapr-operator", "dapr-sentry", "dapr-placement", "dapr-sidecar-injector"}
+)
+
+// StatutOutput represents the status of a named Dapr resource.
+type StatusOutput struct {
+	Name      string `csv:"NAME"`
+	Namespace string `csv:"NAMESPACE"`
+	Healthy   string `csv:"HEALTHY"`
+	Status    string `csv:"STATUS"`
+	Version   string `csv:"VERSION"`
+	Age       string `csv:"AGE"`
+	Created   string `csv:"CREATED"`
+}
+
+// List status for Dapr resources.
+func Status() ([]StatusOutput, error) {
+	client, err := Client()
+	if err != nil {
+		return nil, err
+	}
+
+	var wg sync.WaitGroup
+	wg.Add(len(controlPlaneLabels))
+
+	m := sync.Mutex{}
+	statuses := []StatusOutput{}
+
+	for _, lbl := range controlPlaneLabels {
+		go func(label string) {
+			p, err := client.CoreV1().Pods(v1.NamespaceAll).List(v1.ListOptions{
```

Comment:
As I know dapr control plane will always be in dapr-system, so we can just pass namespace filter "dapr-system" and optimize..

Reply:
It's configurable, we can't expect it to always be on `dapr-system`.

Reply:
ok. In that case, can we take the namespace as optional parameter and use that filter if provided else use NameSpace.All... ?

Reply:
The API server query is very cheap, I think the user experience here not having to specify the namespace is better here, for devs or people who are not aware of Kubernetes specific things.

Reply:
sounds good.